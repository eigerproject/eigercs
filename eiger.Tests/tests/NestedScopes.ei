let x = 5
let y = 0

if x ?= 5 then
	y = 10

	if y ?= 10 then
		let z = 15
		emit(z)
	end
end

emit(y)
emitln(x)

~ ---------------------- ~

x = 5

if x ?= 5 then
	y = 10

	if y ?= 10 then
		x = 20
		emit(x)
	end

	emit(y)
end

emitln(x)