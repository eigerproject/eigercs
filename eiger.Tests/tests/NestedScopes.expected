15105
201020